portals/publisher/src/main/webapp/services/login/idp.jsp

```diff
@@ -0,0 +1,188 @@
+<%--
+  ~ Copyright (c) 2017, WSO2 LLC (http://www.wso2.org) All Rights Reserved.
```

Comment:
We have LLC licence in some files while some others have Inc. Should we make it consistent?

Reply:
+1 sorry for that….